tmk_core/rules.mk

```diff
@@ -426,9 +431,13 @@ check-size:
 		    fi \
 		fi \
 	fi
+check-md5:
+	md5 $(BUILD_DIR)/$(TARGET).hex
```

Comment:
In MSYS2 (and thus possibly Arch) this is actually `md5sum`.

Reply:
Done.

However, I don't know the proper place to determine the command name for md5sum.